public/index.html

```diff
@@ -1,19 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>TITLE</title>
-        <link rel="stylesheet" href="styles.css" />
-        <script src="https://kit.fontawesome.com/cfb22d64e1.js" crossorigin="anonymous"></script>
-    </head>
 
-    <body>
-        <div id="root">
-        </div>
-        <script>
-            var exports = {};
-        </script>
-        <script type="commonjs" src="../client/index.tsx"></script>
-    </body>
-</html>
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GraphQLGateway</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script src="https://kit.fontawesome.com/cfb22d64e1.js" crossorigin="anonymous"></script>
```

Comment:
Question for Milos, I added this script to get the menu icon for the dashboard toolbar. We aren't using it anymore, is this script still needed?

Reply:
I can see your using the settings icon and need this script.

Reply:
Correct, I added this in before seeing Milos's changes for the icon. I can look into alternatives if needed. I just used it since it's what was being used at the moment.